app/src/organisms/ProtocolsLanding/EmptyStateLinks.tsx

```diff
@@ -34,13 +33,18 @@ export function EmptyStateLinks(props: Props): JSX.Element | null {
       paddingBottom={SPACING.spacing3}
       width="96.5%"
     >
-      <StyledText role="complementary" as="h5" marginBottom={SPACING.spacing3}>
+      <StyledText
+        role="complementary"
+        as="label"
```

Comment:
According to the design the text p-semiBold.
```suggestion
        as="p"
```

Reply:
Corrected.